environment.yml

```diff
@@ -27,7 +27,7 @@ dependencies:
     # Dev dependencies (style checks)
     - codespell
     - pre-commit
-    - ruff>=0.9.6
+    - ruff>=0.10.0
```

Comment:
Ruff 0.10.0 isn't available on conda-forge yet, wait for PR at https://github.com/conda-forge/ruff-feedstock/pulls before merging.

Reply:
https://github.com/conda-forge/ruff-feedstock/pull/260 is merged now :rocket: